SPECS/ant/ant.spec

```diff
@@ -1,23 +1,70 @@
+%define _prefix /var/opt/apache-%{name}
+%define _bindir %{_prefix}/bin
+%define _libdir %{_prefix}/lib
 Summary:        Apache Ant
 Name:           ant
 Version:        1.10.9
-Release:        1%{?dist}
-License:        ASL 2.0 and BSD and W3C
-URL:            https://ant.apache.org
-Group:          Applications/System
+Release:        2%{?dist}
+License:        ASL 2.0 AND BSD AND W3C
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-BuildArch:      noarch
```

Comment:
so it is no longer a noarch rpm, isn't it?

Reply:
The linter moved the `BuildArch` below the provides